misc/bazel/rust.bzl

```diff
@@ -0,0 +1,23 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+load("@semmle_code//buildutils-internal:glibc_symbols_check.bzl", "glibc_symbols_check")
+load("@semmle_code//buildutils-internal:lipo.bzl", "universal_binary")
```

Comment:
not in the context of this PR, but we probably should think about moving this rule in this repo or go with the separate bazel lib repo idea we had